pylib/gyp/__init__.py

```diff
@@ -494,7 +494,7 @@ def gyp_main(args):
                 home_vars.append("USERPROFILE")
             for home_var in home_vars:
                 home = os.getenv(home_var)
-                if home != None:
+                if home is not None:
```

Comment:
Let's change this to `if home:` because other Falsey values are just as bad as `None` and we treat `home` as a `str` on the following line.